src/prefect/cli/deploy.py

```diff
@@ -734,6 +745,14 @@ async def _run_single_deploy(
 
     await _create_deployment_triggers(client, deployment_id, triggers)
 
+    if sla_specs := _gather_deployment_sla_definitions(
+        options.get("sla"), deploy_config.get("sla")
```

Comment:
It looks like the `sla` field in a `prefect.yaml` file could be a list. Should the field be named `slas`?

Reply:
@aaazzam expressed a preference for the `sla` name -- I find `slas` to be a little funky myself. I think I'd go with either `sla` or `service_level_agreements`

Reply:
Yeah, pluralizing an acronym is funky. I'd prefer `sla` over `service_level_agreements`.